src/applicants/applicantsController.js

```diff
@@ -55,4 +55,20 @@ const deleteApplicant = async (req, res, next) => {
   }
 };
 
-export default { getAllApplicants, deleteApplicant };
+const restoreApplicant = async (req, res, next) => {
+  const { personId } = req.body;
+  try {
+    const applicant = await Applicant.findByPk(personId);
+    const status = await Status.findByPk(1);
```

Comment:
- You don't have to bother fetching it like this. Usually, when you have statuses, make constants files somewhere and add like this:
```
export const USER_STATUSES: {
    deleted: {
       status_id: 1
   }
}
```

This way you can just use the statuses from the constants without fetching them because they shouldn't be changed at any time in the DB.

Reply:
I added the constants for statuses and updated the files where I fetched statuses to use constants now